capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require './baseDecorator'
+
+class CapitalizeDecorator < BaseDecorator
+  def correct_name
+    @nameable.correct_name.upcase
```

Comment:
As per the requirement, we only need to capitalize the name and not return the whole name in uppercase. Kindly replace the `upcase` with `capitalize`.